minima.go

```diff
@@ -98,11 +98,23 @@ func New() *Minima {
  */
 func (m *Minima) Listen(addr string) error {
 	if m.started {
-		log.Panicf("Minimia's instance is already running at %s.", m.server.Addr)
+		log.Panicf("Minima's instance is already running at %s.", m.server.Addr)
 	}
 	m.server = &http.Server{Addr: addr, Handler: m}
 	m.started = true
-	fmt.Printf(banner, version, addr)
+
+	banner := fmt.Sprintf(`	
+%s  __  __  _  __  _  _  __  __   ____  
+%s |  \/  || ||  \| || ||  \/  | / () \ 
+%s |_|\/|_||_||_|\__||_||_|\/|_|/__/\__\ %s
+%s The Go framework to scale
+%s___________________________________________
+%s Server started at port %s %v
+%s                                                                      
+`, green, green, green, version, blue, red, blue, yellow, addr, reset)
+
+	fmt.Println(banner)
+
```

Comment:
at this point we can just extract this logic in a different function and this has not much to do with listen method

Reply:
yea keep it as constant